tests/utils/test_databricks_utils.py

```diff
@@ -200,3 +202,15 @@ def test_databricks_params_throws_errors(ProfileConfigProvider):
     ProfileConfigProvider.return_value = mock_provider
     with pytest.raises(Exception):
         databricks_utils.get_databricks_host_creds()
+
+
+def test_is_in_databricks_runtime(tmpdir):  # pylint: disable==unused-argument
```

Comment:
```suggestion
def test_is_in_databricks_runtime():
```

If we don't need `tmpdir`, we can remove it.